src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java

```diff
@@ -1065,7 +1065,7 @@ public static void buildPlayersGamesTable(TableView<GamePlayerStatsFX> tableView
         tableView.getColumns().add(rankedColumn);
         extractors.put(rankedColumn, gamePlayerStatsFX -> gamePlayerStatsFX.getGame().getValidity());
 
-        TableColumn<GamePlayerStatsFX, Number> beforeGameRatingColumn = new TableColumn<>("Rating Before Game");
+        TableColumn<GamePlayerStatsFX, Integer> beforeGameRatingColumn = new TableColumn<>("Rating Before Game");
```

Comment:
I'm sure you want to keep the number. In some combinations (I don't exactly remember) you get super weird errors and it's related to not using number.

Reply:
Would that have been because we were using an IntegerProperty and then binding it to a floatProperty.

Now it is an ObjectProperty<Integer> since it can be nullable. I would suspect this removes the issue.